.github/workflows/continuous-integration.yml

```diff
@@ -98,7 +98,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        bucket: [1, 2, 3, 4, 5, 6, 7, 8]
+        bucket: [1, 2, 3, 4, 5, 6, 7, 8, 9]
```

Comment:
Sorry for causing you a merge conflict, but you'll need to renumber your additional test bucket and append a `10` to this list.